src/main/java/org/breedinginsight/brapps/importer/services/processors/GermplasmProcessor.java

```diff
@@ -114,46 +114,47 @@ public GermplasmProcessor(BrAPIGermplasmService brAPIGermplasmService, DSLContex
     public void getExistingBrapiData(List<BrAPIImport> importRows, Program program) throws ApiException {
 
         // Get all of our objects specified in the data file by their unique attributes
-        Map<String, Boolean> germplasmDBIDs = new HashMap<>();
+        Map<String, Boolean> germplasmAccessionNumbers = new HashMap<>();
         for (int i = 0; i < importRows.size(); i++) {
             BrAPIImport germplasmImport = importRows.get(i);
-            if (germplasmImport.getGermplasm() != null) {
+            Germplasm germplasm = germplasmImport.getGermplasm();
+            if (germplasm != null) {
 
                 // Retrieve parent dbids to assess if already in db
-                if (germplasmImport.getGermplasm().getFemaleParentDBID() != null) {
-                    germplasmDBIDs.put(germplasmImport.getGermplasm().getFemaleParentDBID(), true);
+                if (germplasm.getFemaleParentAccessionNumber() != null) {
+                    germplasmAccessionNumbers.put(germplasm.getFemaleParentAccessionNumber(), true);
                 }
-                if (germplasmImport.getGermplasm().getMaleParentDBID() != null) {
-                    germplasmDBIDs.put(germplasmImport.getGermplasm().getMaleParentDBID(), true);
+                if (germplasm.getMaleParentAccessionNumber() != null) {
+                    germplasmAccessionNumbers.put(germplasm.getMaleParentAccessionNumber(), true);
                 }
-                if (germplasmImport.getGermplasm().getAccessionNumber() != null) {
-                    germplasmDBIDs.put(germplasmImport.getGermplasm().getAccessionNumber(), false);
+                if (germplasm.getAccessionNumber() != null) {
+                    germplasmAccessionNumbers.put(germplasm.getAccessionNumber(), false);
                 }
 
                 //Retrieve entry numbers of file for comparison with parent entry numbers
-                if (germplasmImport.getGermplasm().getEntryNo()!= null) {
-                    germplasmIndexByEntryNo.put(germplasmImport.getGermplasm().getEntryNo(), i);
+                if (germplasm.getEntryNo()!= null) {
+                    germplasmIndexByEntryNo.put(germplasm.getEntryNo(), i);
                 }
 
-                Integer count = fileGermplasmByName.getOrDefault(germplasmImport.getGermplasm().getGermplasmName(), 0);
-                fileGermplasmByName.put(germplasmImport.getGermplasm().getGermplasmName(), count+1);
+                Integer count = fileGermplasmByName.getOrDefault(germplasm.getGermplasmName(), 0);
+                fileGermplasmByName.put(germplasm.getGermplasmName(), count+1);
             }
         }
 
         // If parental DBID, should also be in database
-        existingGermplasms = new ArrayList<>();
```

Comment:
Previous comment should probably say GID or accession number instead of DBID

Reply:
Done, I also fixed other, similar comments that referenced DBID erroneously. https://github.com/Breeding-Insight/bi-api/pull/400/commits/5c7b529e76aaa6d50aab4b7b6e38d78f71d79af4